### ISSUE TITLE
TDRD-596

### ISSUE THREAD

**Issue by ian-hoyle** [nationalarchives/tdr-transfer-frontend#4646]
Accessibility Testing Metadata Upload
Context

Changes to user journey should be accessibility tested as part of the definition of done for each ticket. However given the launch of a significant new user journey, it would be worthwhile to do a pass through the final journey to review for any accessibility issues that may have been missed. 

Requirements

Review new metadata upload pages for any accessibility issues

Refer to: 

[https://github.com/nationalarchives/tdr-dev-documentation/blob/60b786ae0be0c0a48c8[…]fd64f4656ec7b3/manual/development-process/test-accessibility.md](https://github.com/nationalarchives/tdr-dev-documentation/blob/60b786ae0be0c0a48c83b2f83efd64f4656ec7b3/manual/development-process/test-accessibility.md)

The draft-metadata pages are created from the scala play html templates that can be found in this directory
https://github.com/nationalarchives/tdr-transfer-frontend/tree/master/app/views/draftmetadata

Please do the work in a branch in the  tdr-transfer-fronted repo
